### PR TITLE
Vet Clinic : Animal Table

### PR DESCRIPTION
**Pull request by jeanbuhendwa**
#### In this pull request, I have added the animal table and inserted some animals into it. 

- [ ] all animals whose name ends in "mon"
Screenshot 2023-05-01 at 18 06 40

- [ ] List the name of all animals born between 2016 and 2019.
Screenshot 2023-05-01 at 18 10 30

- [ ] List the names of all neutered animals with less than 3 escape attempts.
Screenshot 2023-05-01 at 18 14 04

- [ ] List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot 2023-05-01 at 18 16 31

- [ ] List name and escape attempts of animals that weigh more than 10.5kg
Screenshot 2023-05-01 at 18 17 54

- [ ] Find all animals that are neutered.
Screenshot 2023-05-01 at 18 19 17

- [ ] Find all animals not named Gabumon.
Screenshot 2023-05-01 at 18 22 09

- [ ] Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
Screenshot 2023-05-01 at 18 24 47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,30 +30,12 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 [your_project_name] <a name="about-project"></a>
```
Comment (rotimiazeez) [inline README.md]: Kindly ensure this is your project name👍🏼

File: README.md [hunk 2]
```diff
@@ -248,44 +94,20 @@ Feel free to check the [issues page](../../issues/).
...
-  - [Answer_2]
+I would like to thank Microverse team for providing this project.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (rotimiazeez) [inline README.md:111]: The provided MIT License link is broken 😢. Kindly fix this 👍🏼 
Screenshot_20230501_165702

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @jeanbuhendwa 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Correct query results ✔️
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (jeanbuhendwa) [reply README.md:111]: Thanks @rotimiazeez for the feedback.

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @jeanbuhendwa  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All project requirements met :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**